docker/etna-cpu/requirements.txt

```diff
@@ -1,2 +1,2 @@
 jupyterlab
```

Comment:
Should we delete these files?

Reply:
If we are going to include a jupyter extension into our docker image, then we should remove it. Otherwise, we shouldn't.

Reply:
We have discussed it.
And we've ended on that jupyterlab is already in all-dev extras.
If it's not that case we should add it to poetry too.